scripts/generate-graph.ts

```diff
@@ -4,7 +4,7 @@ import { join } from 'path';
 import * as yargs from 'yargs';
 import { ensureDirSync, statSync } from 'fs-extra';
 
-async function generateGraph(directory: string, name: string) {
```

Comment:
This function should probably be async. The IIFE at the bottom of this function is an async function but is not awaited. Could you add an await to that function call?